code/modules/client/preferences.dm

```diff
@@ -1394,10 +1401,15 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					if(pickedPDAStyle)
 						pda_style = pickedPDAStyle
 				if("pda_color")
-					var/pickedPDAColor = input(user, "Choose your PDA Interface color.", "Character Preference",pda_color) as color|null
+					var/pickedPDAColor = input(user, "Choose your PDA Interface color.", "Character Preference", pda_color) as color|null
 					if(pickedPDAColor)
 						pda_color = pickedPDAColor
 
+				if("phobia")
+					var/phobiaType = input(user, "What are you scared of?", "Character Preference", phobia) as null|anything in SStraumas.phobia_types
+					if(phobiaType)

```

Comment:
I'd attach an else here, that way they can pick again if something goes wrong.

Reply:
On `if(phobiaType)`?

Reply:
Yes.

Reply:
They can just press the button again if it doesn't work.

Reply:
But there's no indication that it didn't work, unless you're really paying attention.